test/feeder_ex_test.exs

```diff
@@ -30,6 +30,11 @@ defmodule FeederExTest do
 
   end
 
+  test "parse! returns just the parsed results on success" do
+    parsed_feed = FeederEx.parse!(File.read!(@sample_file))
+    assert parsed_feed.title == "Liftoff News"
+  end
```

Comment:
Note, I will of course remove the WIP commit once this is through and squash everything down. It just seems the easiest way to show you what I've tried.

Apologies for my newness at Elixir! thanks for your patience & most useful library :)